Linting fixes

saabi opened the pull request:
Closes #400

mindrones reviewed (approved):
GTM after adding changelogs

saabi committed:
Linting fixes

- Fixed glob patterns in linting script invocations
- Linting now fails on warnings
- Disabled a11y warning

Closes #400
